.editorconfig

```diff
@@ -0,0 +1,64 @@
+
```

Comment:
I dont' want to look into the file. Let's keep it and see if it breaks some our code-style rules later.

Reply:
Ok